tsconfig.json

```diff
@@ -12,5 +12,6 @@
     "moduleResolution": "node10",             /* NOTE(AR) just making it explicit that this is the tsc compilation strategy for module resolution that we are currently using */
     "rootDir": "src",
     "outDir": "dist",                         /* Redirect output structure to the `dist/` directory. */
+    "resolveJsonModule": true
```

Comment:
Since we are no longer importing `json`, this flag is not needed.

Reply:
@marmoure Sure it does, we are importing JSON.

Reply:
@plutonik-a I just doubled checked, I don't see any json import in your changes, please point it to me.